app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -26,6 +26,15 @@ module Form526ClaimFastTrackingConcern
   CLAIM_REVIEW_BASE_CODES = %w[030 040].freeze
   CLAIM_REVIEW_TYPES = %w[higherLevelReview supplementalClaim].freeze
 
+  def claim_age_in_days(pending_ep)
```

Comment:
Pulled this out into a new method to prevent `def prepare_for_ep_merge!` from getting too long.